### ISSUE TITLE
vue-property-decorator dependency problem

### ISSUE THREAD

**Issue by rudolfbyker** [Bartozzz/vue-pwa-install#16]
To reproduce:

- In an existing Vue CLI 3 project,
- run `yarn add vue-pwa-install`,
- add `import VueInstall from "vue-pwa-install";` to `main.js`
- try to build with `vue-cli-service build`

```
⠙  Building for production...

 ERROR  Failed to compile with 1 errors                                                                                                                                                                                           11:38:34 AM

This dependency was not found:

* vue-property-decorator in ./node_modules/vue-pwa-install/lib/index.js

To install it, you can run: npm install --save vue-property-decorator
```

Running `yarn add vue-property-decorator` fixes it, but I don't want that as a direct dependency of my project.

I see that `vue-property-decorator` is listed under `devDependencies` in your project. Should it not be moved up to `dependencies`?

**Comment by Bartozzz** [comment]
Thanks for reporting – I'll have a look at this today. Moving this dependency to `dependencies` should probably resolve the issue. 😄

**Comment by Bartozzz** [comment]
This should be resolved in [`v1.1.0`](https://www.npmjs.com/package/vue-pwa-install). You can have a look at [`.codesandbox/sandbox`](https://github.com/Bartozzz/vue-pwa-install/tree/master/.codesandbox/sandbox) for examples.
